tox.ini

```
@@ -12,12 +12,13 @@ deps =
     pytest-cov
     pytest-sanic
     pytest-sugar
+    pytest-benchmark
+    pytest-dependency
```

Why are we adding this dependency?

---

pytest-benchmark already was thers - I just moved it up.

pytest-dependency
Running
&emsp;test_loaded_module_from_file_location_name
and
&emsp;test_update[from_file]
makes seanse only when
&emsp;test_load_module_from_file_location
succed, so I had to mark first two tests deependent on last one.

---

Is that OK ?

---

I'd rather not be adding any additional dependencies right now.

---

We can not use pytest-dependency, but then we need to resign from marking
    test_loaded_module_from_file_location_name
and
    test_update[from_file]
to run only when 
    test_load_module_from_file_location
succeed.

So in other words if 
    test_load_module_from_file_location
fail, then
    test_loaded_module_from_file_location_name
and
    test_load_module_from_file_location
will also fail instead of just being skipped.